jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-upgrade.yaml

```diff
@@ -0,0 +1,112 @@
+---
+- job:
+    name: openshift-cluster-integreatly-upgrade
```

Comment:
I think this should start with `osd` prefix.

Reply:
well osd-cluster-integreatly-upgrade already exists, should I name it openshift-cluster-integreatly-upgrade-test or install-upgrade?